### PR TITLE
Adds a "No Prayers" Screen

### PR DESCRIPTION
**Pull request by richarddubay**
### What does this PR do, or why is it needed?
This PR adds a "No Prayers" section to the prayer list so that if there are no prayers for a particular menu group, it will show this instead of a blank screen.

### How do I test this PR?
Run the sim.
Go the the "My Community" tab.
Check out the new "No Prayers" screen.

Screenshot 2019-05-13 12 49 00

---
> I am affirming this is my _best_ work ([Ecclesiastes 9:10](https://www.bible.com/bible/97/ECC.9.10.MSG))

## TODO

- [x] PR has a relevant title that will be understandable in a public changelog (ie...non developers)
- [x] Closes [DEV-3755](https://newspring.atlassian.net/browse/DEV-3755)
- [x] No new warnings in tests, in storybook, and in-app
- [x] Upload GIF(s) of iOS and Android if applicable
- [x] Set two relevant reviewers

## REVIEW

**Manual QA**

- [ ] Manual QA on iOS and ensure it looks/behaves as expected
- [ ] Manual QA on Android and ensure it looks/behaves as expected

**Code Review: Questions to consider**

- [ ] Read through the "Files changed" tab _very carefully_
- [ ] Edge cases: what assumptions are made about input?
- [ ] What kind of tests could be written?
- [ ] How might we make this easier for someone else to understand?
- [ ] Could the code be simpler?
- [ ] Will the code be easy to modify in the future?
- [ ] What's one part of these changes that makes you excited to merge it?

---

> The purpose of PR Review is to _improve the quality of the software._

### CODE REVIEW

File: packages/newspringchurchapp/src/prayer/PrayerList/index.js [hunk 1]
```diff
@@ -109,72 +133,82 @@ class PrayerList extends PureComponent {
...
+                    name: prayer.firstName,
+                  }))}
+                  scrollEnabled={false}
+                />
+              ) : (
+                <StyledView>
+                  <StyledH3View>
+                    <H3>No Prayers!</H3>
+                  </StyledH3View>
+                  <BodyText>{noPrayerText}</BodyText>
```
Comment (redreceipt) [inline packages/newspringchurchapp/src/prayer/PrayerList/index.js]:
```suggestion
                  <BodyText>{`There are currently no prayers for your ${type}. Go back and add one!`}</BodyText>
```

File: packages/newspringchurchapp/src/prayer/PrayerList/index.js [hunk 2]
```diff
@@ -86,20 +107,23 @@ class PrayerList extends PureComponent {
...
     }
-    return { query, prayers, variables };
+    return { query, prayers, type, variables };
   };
 
   render() {
-    const { query, prayers, variables } = this.calculateQuery();
+    const { query, prayers, type, variables } = this.calculateQuery();
     const { navigation } = this.props;
+    const noPrayerText = `There are currently no prayers for your ${type}. Go back and add one!`;
```
Comment (redreceipt) [inline packages/newspringchurchapp/src/prayer/PrayerList/index.js]:
```suggestion
```

**Review by redreceipt — DISMISSED** [review]
I think does what we need it to. For future, do you think this should happen in place of the `PrayerPreviewCard` on the main menu screen?

File: packages/newspringchurchapp/src/prayer/PrayerList/index.js (hunk 1 above)
Comment (redreceipt) [reply packages/newspringchurchapp/src/prayer/PrayerList/index.js]: the variable here is prob unnecessary here since we're just hardcoding a few lines up

**Review by redreceipt — DISMISSED** [review]

File: packages/newspringchurchapp/src/prayer/PrayerList/index.js (hunk 1 above)
Comment (richarddubay) [reply packages/newspringchurchapp/src/prayer/PrayerList/index.js]: Haha! I tried this first and couldn't get it to work so that's why I made the variable. I think I was missing the `{}` around it the first time. I totally added this. Great catch! 🎉

**Comment by richarddubay** [comment]
Ooh ... that's an interesting thought about the `PrayerPreviewCard`. Yeah, if there are no prayers for that menu item and we're showing whatever the first prayer for that thing is, then I think it should definitely show _something_ to encourage them to add one.

**Comment by redreceipt** [comment]
conflicts, womp womp :)

**Review by redreceipt — APPROVED** [review]
🤘
